app/uk/gov/hmrc/preferencesadminfrontend/services/MessageService.scala

```diff
@@ -51,4 +56,35 @@ class MessageService @Inject()(messageConnector: MessageConnector) {
           case _ => Right(response.body)
       })
 
+  type ResponseStatus = Int
+  type ResponseBody = String
+  type MessageId = String
+
+  val messageContent = Base64.encodeBase64String(penaltyChargeApologies().toString().getBytes("UTF-8"))
+
+  def sendPenalyChargeApplologyMessage(email: String, sautr: String)(
```

Comment:
spelling- sendPenaltyChargeApologyMessage

Reply:
updated the PR